RELEASE 1.0.0 (#26)
nf-core/nanostring: docs/usage.md

## Introduction

The **nf-core/nanostring** pipeline allows the analysis of NanoString data. The pipeline performs quality control, normalization and annotation of the obtained counts.

## Samplesheet input

You will need to create a samplesheet with information about the samples you would like to analyse before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row as shown in the examples below.

```bash
--input '[path to samplesheet file]'
```

### Multiple runs of the same sample

The `sample` identifiers should be the same when you measured the same sample multiple times. Below is an example for the same sample measured twice:

```console
RCC_FILE,SAMPLE_ID
/path/to/sample1.RCC,sample1
/path/to/sample2_1.RCC,sample2
/path/to/sample2_2.RCC,sample2
```

### Full samplesheet

> The samplesheet can have as many columns as you desire, however, there is a strict requirement for the first two columns to match those defined in the table below.

Reviewer: ```suggestion
The samplesheet can have as many columns as you desire, however, there is a strict requirement for the two columns `RCC_FILE` and `SAMPLE_ID`.
```
